src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -207,62 +181,27 @@ public void RequestAnalysis(string path, string charset, IEnumerable<AnalysisLan
             }
         }
 
-        public void AddSinkManager(SinkManager manager)
-        {
-            lock (managers)
-            {
-                managers.Add(manager);
-
-                foreach (var issueTracker in issueTrackers)
-                {
-                    manager.AddFactory(issueTracker.Factory);
-                }
-            }
-        }
-
-        public void RemoveSinkManager(SinkManager manager)
-        {
-            lock (managers)
-            {
-                managers.Remove(manager);
-            }
-        }
-
         public void AddIssueTracker(IIssueTracker issueTracker)
         {
-            lock (managers)
+            lock (issueTrackers)
```

Comment:
This class no longer manages the sink managers, so it now locks on the issue trackers list instead.